src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -1,8 +1,8 @@
 @import "scss/constants";
 
 $sand: #937c63;
-$white: #ffffff;
 $border-radius: 28px;
+$avatarpulse-box-shadow-color: var(--primary-accent--under-65a);
```

Comment:
[OOS]

A musing on the notion if there could be a constant like `$primary--under-65a` for this one.
Is it possible/viable to generate constants for `var()` strings even?

Reply:
could be in a follow-up PR, not sure about generating consts for `var()`